data/mods/Magiclysm/items/spell_scrolls.json

```diff
@@ -1986,5 +1986,32 @@
     "name": { "str": "Scroll of Machine Mind", "str_pl": "Scrolls of Machine Mind" },
     "description": "Gain an intuitive understanding of computers and electronics, allowing an easier time repairing or using them.",
     "use_action": { "type": "learn_spell", "spells": [ "technomancer_gain_electronics_computer_spell" ] }
+  },
+  {
+    "type": "BOOK",
+    "copy-from": "spell_scroll",
+    "id": "spell_scroll_kelvinist_heat_food",
+    "//": "Kelvinist spell",
+    "name": { "str": "Scroll of Baking Hands", "str_pl": "Scrolls of Baking Hands" },
+    "description": "Warm up your food using pure mana.",
```

Comment:
+Charcoal isn't pure mana 🤔

Reply:
The charcoal fuels the spell, not the heating--you're not actually burning the charcoal, just using its sympathetic connection to cooking.